addon/components/power-select.js

```diff
@@ -569,6 +587,13 @@ export default Component.extend({
   }).restartable(),
 
   // Methods
+  focusInput() {
+    if(this.get('multiSelect')) {
+      let input = document.querySelector(`#ember-power-select-trigger-multiple-input-${this.get('publicAPI.uniqueId')}`);
+      run.next(() => input && document.activeElement !== input && input.focus());
```

Comment:
```suggestion
     input &&  run.next(() => document.activeElement !== input && input.focus());
```
Let's not queue an action to runloop at all, when there is not input element present.